src/main/scala/mesosphere/marathon/api/akkahttp/EntityMarshallers.scala

```diff
@@ -119,22 +156,33 @@ object EntityMarshallers {
   implicit val loggerChangeUnmarshaller = playJsonUnmarshaller[LoggerChange]
   implicit val stringMapMarshaller = playJsonMarshaller[Map[String, String]]
   implicit val pluginDefinitionsMarshaller = playJsonMarshaller[PluginDefinitions]
+  implicit val deploymentResultMarshaller = playJsonMarshaller[Messages.DeploymentResult]
+  implicit val enrichedTaskMarshaller = playJsonMarshaller[EnrichedTask]
```

Comment:
Ideally this is `raml.EnrichedTask`. I'm currently working on this in https://github.com/mesosphere/marathon/pull/5556. So I think it is fine here.